.gitignore

```diff
@@ -10,6 +10,9 @@ web-app/src/proto/*
 api/proto/*
 !api/proto/.gitkeep
 
+# Generated CSS framework files
+web-app/src/assets/css/tailwind.css
```

Comment:
If you want to do it here rather than `web-app/.gitignore`, I'd rather just move everything from that one to here so that there's only one place to look for ignored files within `web-app`

Reply:
Oh! Didn't even see that one. Definitely in favor of merging them into a global one for better visibility.

There's probably a similar discussion about the directory-specific README files and how much stuff belongs at the top-level.